tests/mocha/navigation_test.js

```diff
@@ -401,7 +401,7 @@ suite('Navigation', function() {
         Blockly.navigation.setCursor(this.workspace.cursor);
         Blockly.mainWorkspace = this.workspace;
         this.fieldBlock1 = this.workspace.newBlock('field_block');
-        Blockly.keyboardAccessibilityMode = true;
+        Blockly.keyboardAccessibility = true;
```

Comment:
This variable doesn't exist anywhere else. Where did it come from?

Reply:
thin air :/ should be fixed now